attoparsec.cabal

```diff
@@ -6,7 +6,7 @@ category:        Text, Parsing
 author:          Bryan O'Sullivan <bos@serpentine.com>
 maintainer:      Bryan O'Sullivan <bos@serpentine.com>, Ben Gamari <ben@smart-cactus.org>
 stability:       experimental
-tested-with:     GHC == 7.4.2, GHC ==7.6.3, GHC ==7.8.4, GHC ==7.10.3, GHC ==8.0.2, GHC ==8.2.2, GHC==8.4.4, GHC==8.6.5, GHC==8.8.1, GHC==8.10.1
+tested-with:     GHC == 7.4.2, GHC ==7.6.3, GHC ==7.8.4, GHC ==7.10.3, GHC ==8.0.2, GHC ==8.2.2, GHC==8.4.4, GHC==8.6.5, GHC==8.8.1, GHC==8.10.1, GHC == 9.0.1
```

Comment:
- Change `tested-with`: 8.8.1 --> 8.8.4 and 8.10.1 --> 8.10.4 
- run `haskell-ci regenerate`
- commit `.github/*`

Reply:
> Strangely enough 0.13.2.4 builds fine with GHC 9.0.1, but not the latest release, 0.13.2.5.

Because 0.13.2.4 doesn't depend on `ghc-prim`.

Reply:
@andreasabel, I made the suggested changes. Thank you for your feedback.